container_service_extension/configure_cse.py

```diff
@@ -1162,7 +1203,7 @@ def _upgrade_to_35(client, config, ext_vcd_api_version,
                    admin_password, msg_update_callback=utils.NullPrinter(),
                    log_wire=False):
     # Update amqp exchange
-    _create_amqp_exchange(
+    _create_amqp_exchange(  # TODO: check if using mqtt
```

Comment:
So this TODO Will get cleaned up as part of same release or later?
Lets have offline discussion on this :)